charts/back8sup/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: back8sup
 description: Deploy back8sup to a Kubernetes Cluster
 type: application
-version: 0.2.1
```

Comment:
I would bump to 0.3.0, because we increase by minor version in the `appVersion` and switch the docker repository, which could also cause issues for users.

Reply:
you're right.